components/notifier/tests/test_notification.py

```diff
@@ -0,0 +1,27 @@
+""""Unit tests for the notification."""
+
+import unittest
+
+from notification import Notification
+
+
+class StrategiesTestCase(unittest.TestCase):
```

Comment:
Why is this called "StrategiesTestCase", it's just testing the Notification class, right?

```suggestion
class NotificationTestCase(unittest.TestCase):
```